gammapy/data/ivoa.py

```diff
@@ -0,0 +1,327 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+import logging
+import os
+import numpy as np
+from astropy.table import Column, Table
+from gammapy.data import DataStore
+from gammapy.utils.scripts import make_path
+
+__all__ = ["to_obscore_table"]
+
+
+log = logging.getLogger(__name__)
+log.setLevel(logging.INFO)
+
+
+def _obscore_def():
+    """Generate the Obscore default table
+    In case the obscore standard changes, this function should be changed consistently
+
```

Comment:
Can you add a link towards the documentation of the obsscore format?

Reply:
sure